mcp/src/tools/audience-tools.ts

```diff
@@ -16,40 +23,9 @@ export class AudienceTools extends CourierMcpTools {
       }
     );
 
-    // // Create or update an audience
-    // this.mcp.tool(
-    //   'update_audience',
-    //   'Create or update an audience by its ID',
-    //   {
-    //     audience_id: z.string(),
-    //     name: z.string().optional(),
-    //     description: z.string().optional(),
-    //     filter: z.any().optional(),
-    //   },
-    //   async ({ audience_id, name, description, filter }) => {
-    //     const request: any = {};
-    //     if (name !== undefined) request.name = name;
-    //     if (description !== undefined) request.description = description;
-    //     if (filter !== undefined) request.filter = filter;
-    //     return await this.mcp.client.audiences.update(audience_id, request);
-    //   }
-    // );
-
-    // // Delete an audience by audience_id
-    // this.mcp.tool(
-    //   'delete_audience',
-    //   'Delete an audience by its ID',
-    //   {
-    //     audience_id: z.string(),
-    //   },
-    //   async ({ audience_id }) => {
-    //     return await this.mcp.client.audiences.delete(audience_id);
-    //   }
-    // );
-
     // List members of an audience
-    this.mcp.tool(
-      'list_audience_members',
+    this.registerToolIfNeeded(
+      AudienceTools.tools[1],
```

Comment:
a thought - using a map instead of array would avoid indexing on the array position. and then when you need to register everything could do `AudienceTools.tools.values()`